pkg/minishift/registration/redhat.go

```diff
@@ -151,3 +152,9 @@ func (registrator *RedHatRegistrator) isRegistered() (bool, error) {
 		return false, nil
 	}
 }
+
+func redactPassword(err error) error {
```

Comment:
Should be `func redactPassword(data string) string {`

Reply:
Then return errors.New(`returned string`)

Reply:
picky, but makes it more general